packages/xcm-types/xcm-types.ts

```diff
@@ -69,15 +93,18 @@ type GenericAsset<Id, Fun> = {
   fun: Fun;
 };
 
+export type AssetV5 = GenericAsset<AssetIdV5, FungibilityV5>;
 export type AssetV4 = GenericAsset<AssetIdV4, FungibilityV4>;
 export type AssetV3 = GenericAsset<AssetIdV3, FungibilityV3>;
 export type AssetV2 = GenericAsset<AssetIdV2, FungibilityV2>;
 
 export type AssetId = AssetIdV4;
```

Comment:
The library's point is that it always uses the latest XCM version as "the default way of handling XCM" from a user's perspective.

All the XCM types that don't mention a version should point to the latest version (i.e., V5).

```suggestion
export type AssetId = AssetIdV5;
```